pkg/asset/ignition/machine/master_test.go

```diff
@@ -1,18 +1,21 @@
 package machine
 
 import (
+	"encoding/json"
 	"testing"
 
+	igntypes "github.com/coreos/ignition/config/v2_2/types"
 	"github.com/stretchr/testify/assert"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/utils/pointer"
 
 	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/ignition"
 	"github.com/openshift/installer/pkg/asset/installconfig"
 	"github.com/openshift/installer/pkg/asset/tls"
 	"github.com/openshift/installer/pkg/ipnet"
 	"github.com/openshift/installer/pkg/types"
-	"github.com/openshift/installer/pkg/types/aws"
```

Comment:
I believe we should not be removing something that is existing, and adding our functionality. Any reason to remove the aws packages